vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

```diff
@@ -1,108 +1,105 @@
 
-=== "OpenAI"
 
-    To use OpenAI with Vizro-AI, you must have an account with paid-for credits available. None of the free accounts will suffice.
-
-    - gpt-3.5-turbo `default`
-    - gpt-4-turbo
-    - gpt-4o
-    - gpt-4
-    - gpt-4-0613
-    - gpt-4-1106-preview
-    - gpt-4-turbo-2024-04-09
-    - gpt-4-turbo-preview
-    - gpt-4-0125-preview
-    - gpt-3.5-turbo-1106
-    - gpt-3.5-turbo-0125
-    - gpt-4o-2024-05-13
-    - gpt-4o-mini-2024-07-18
-    - gpt-4o-mini
+### Setting model via string for ease of use
 
-=== "Anthropic"
+We have created shortcuts with sensible defaults (mainly setting `temperature=0`) for some common vendors. These models can be chosen by using the string format in the tabs below. If no model is provided, the default (currently `"gpt-4o-mini"`) is selected.
 
-    :octicons-hourglass-24: In development
-
-=== "MistralAI"
+```py
+vizro_ai = VizroAI(model="<chosen model>")
+```
 
-    :octicons-hourglass-24: In development
+!!!note
+    For the string settings to work, you must provide your API key via environment variables. The relevant variable names to be set are noted in each vendor tab.
 
+=== "OpenAI"
 
-These models offer different levels of performance and cost to Vizro-AI users.
+    | Env variable      | Name(s)                               |
+    | -----------       | ------------------------------------  |
+    | API key           | `OPENAI_API_KEY`                      |
+    | Base API URL      | `OPENAI_API_BASE`                     |
 
-### Chart generation
+    To use OpenAI with Vizro-AI, you must have an account with paid-for credits available. None of the free accounts will suffice. You can check [all available OpenAI models including pricing on their website](https://platform.openai.com/docs/models). This will also explain which version the below string acronyms currently point to.
 
-For basic chart creation with OpenAI, the **gpt-4o-mini** and **gpt-3.5** model series have lower price points and faster speeds, but they do not offer sophisticated charting.
+    - `gpt-4o-mini` **default**
+    - `gpt-4-turbo`
+    - `gpt-4o`
+    - `gpt-4`
+    - `gpt-3.5-turbo`
 
-Consider upgrading to the **gpt-4o** and **gpt-4** model series for more demanding tasks. The downside of using these models is that they come at a higher cost.
+=== "Anthropic"
```

Comment:
Other vendors are not working properly with dashboard yet. We shall note these only works for chart creation now, not for dashboard